Security/Util/String.php

```diff
@@ -5,6 +5,8 @@
 /**
  * String utility functions.
  *
+ * @deprecated This class will break in PHP 7; use hash_equals() instead
```

Comment:
if it is deprecated, you should make sure it is not used anymore in the bundle (keeping it only for BC).

Deprecating a class while still using it in the same package is bad, as the deprecation precisely means it should not be used anymore